Mathlib/CategoryTheory/Limits/ConcreteCategory.lean

```diff
@@ -7,6 +7,8 @@ import Mathlib.CategoryTheory.ConcreteCategory.Basic
 import Mathlib.CategoryTheory.Limits.Preserves.Basic
 import Mathlib.CategoryTheory.Limits.TypesFiltered
 import Mathlib.CategoryTheory.Limits.Yoneda
+import Mathlib.Algebra.Module.LinearMap.Defs
```

Comment:
A consequence of this import is that now the file proving that the category of abelian groups is abelian, now imports a bunch of ring theory. Maybe move this theorem to a new file?